grunt/tasks/_browserify-bundles.js

```diff
@@ -49,30 +41,11 @@ module.exports = {
   'cartodb-specs': {
     src: [
       'test/fail-tests-if-have-errors-in-src.js',
-      'test/spec/cartodb.spec.js',
+      'test/spec/cartodb.spec.js'
     ],
     dest: '<%= config.tmp %>/cartodb-specs.js',
-  },
-
-  'cartodb.mod.torque': {
-    src: 'src/cartodb.mod.torque.js',
-    dest: '<%= config.dist %>/cartodb.mod.torque.uncompressed.js',
-    options: {
-      external: sharedModules
-    }
-  },
-  'cartodb.mod.torque-specs': {
-    src: 'test/spec/cartodb.mod.torque.spec.js',
-    dest: '<%= config.tmp %>/cartodb.mod.torque-specs.js',
-    options: {
-      external: sharedModules
-    }
-  },
-  'cartodb.mod-torque-srcSpecs': {
-    src: torqueSpecs,
-    dest: '<%= config.tmp %>/cartodb.mod.torque-srcSpecs.js',
     options: {
-      external: sharedModules
+      require: sharedModules
```

Comment:
I think we could remove this altogether, since the module loading code was removed, since there's no need to load things in a separate bundle, right?